src/Parser/Hunter/Marksmanship/Modules/Features/Abilities.js

```diff
@@ -90,95 +90,159 @@ class Abilities extends CoreAbilities {
     },
     {
       spell: SPELLS.TRUESHOT,
-      category: Abilities.SPELL_CATEGORIES.COOLDOWNS,
-      getCooldown: (_, combatant) => 180 - combatant.owner.modules.quickShot.traitCooldownReduction,
-      recommendedEfficiency: 1.0,
-    },
+      category:
+      Abilities.SPELL_CATEGORIES.COOLDOWNS,
+      getCooldown:
+        (_, combatant) => 180 - combatant.owner.modules.quickShot.traitCooldownReduction,
```

Comment:
What happened here?

Reply:
Interesting, probably when I played around with the sentinel spell. Nothing is changed